exposed-jdbc/src/main/kotlin/org/jetbrains/exposed/sql/statements/jdbc/JdbcPreparedStatementImpl.kt

```diff
@@ -47,6 +48,19 @@ class JdbcPreparedStatementImpl(
 
     override fun executeUpdate(): Int = statement.executeUpdate()
 
+    override fun executeMultiple(): List<StatementResult> {
+        // execute() returns true only if first result is a ResultSet
+        return if (statement.execute()) {
+            listOf(StatementResult.Object(statement.resultSet))
+        } else {
+            // getMoreResults() returns true only if next result is a ResultSet
+            while (!statement.getMoreResults(Statement.CLOSE_CURRENT_RESULT)) {
+                if (statement.updateCount == -1) return emptyList()
+            }
+            listOf(StatementResult.Object(statement.resultSet))
+        }
+    }
+
```

Comment:
The function is currently only set up to return the first `ResultSet` found in all generated results.
If this feature is extended to enable returning all results, beyond just using `exec()`, this function will need to be refactored accordingly.